src/iso19111/coordinateoperation.cpp

```diff
@@ -5471,15 +5471,15 @@ void Conversion::_exportToPROJString(
     const auto &convName = nameStr();
     bool bConversionDone = false;
     bool bEllipsoidParametersDone = false;
-    bool useETMerc = false;
+    bool useSTMerc = false;
     if (methodEPSGCode == EPSG_CODE_METHOD_TRANSVERSE_MERCATOR) {
         // Check for UTM
         int zone = 0;
         bool north = true;
-        bool etMercSettingSet = false;
-        useETMerc = formatter->getUseETMercForTMerc(etMercSettingSet) ||
-                    l_method->getPrivate()->projMethodOverride_ == "etmerc";
-        if (isUTM(zone, north) && !(etMercSettingSet && !useETMerc)) {
+        bool stMercSettingSet = false;
+        useSTMerc = formatter->getUseSTMercForTMerc(stMercSettingSet) ||
+                    l_method->getPrivate()->projMethodOverride_ == "stmerc";
+        if (isUTM(zone, north) && !(stMercSettingSet && !useSTMerc)) {
```

Comment:
the negation before useSTMerc should likely be removed. If the user forces STMerc, we don't want UTM to be expressed as +proj=utm, but rather +proj=stmerc ...